site/en/docs/extensions/mv3/getstarted/tut-reading-time/index.md

```diff
@@ -0,0 +1,304 @@
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/VczSGe8eh0Xv7nTXxhxg.png", 
+alt="Reading time extension in the extension Welcome page", width="500", height="116", class="screenshot" %}
+  <figcaption>
+  Reading time extension on the extension's Welcome page. 
+  </figcaption>
+</figure>
+
+In this guide, we’re going to explain the following concepts:
+
+- The extension manifest.
+- What icon sizes an extension uses.
+- How to inject code on pages using [content scripts][doc-cs].
+- How to use match patterns.
+- Extension permissions
+
+## Before you start {: #prereq }
+
+This guide assumes that you have basic web development experience. We recommend checking out
+[Development Basics][doc-dev-basics] for an introduction to the extension development workflow.
+
+## Build the extension {: #build }
+
+To start, create a new directory called `Reading time` that will hold the extension's files. If you
+prefer, you can download the complete source code on [Github][github-reading-time].
+
+### Step 1: Add information about the extension {: #step-1 }
+
+The manifest JSON file is the only required extension file. It holds important information about the
+extension. Create a `manifest.json` file in the _root_ of the project and add the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Reading time",
+  "version": "1.0",
+  "description": "Add the reading time to Chrome Extension documentation articles",
+  ...
+}
+```
+
+These keys contain basic metadata for the extension. It appears in the Extension Management page
+and, when published, in the Chrome Web Store listing. To dive deeper, check out the
+[`"name"`][man-name], [`"version"`][man-ver] and [`"description"`][man-desc] keys on the
+[Manifest][doc-manifest] overview page.
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Other facts about the extension Manifest**
+{% endDetailsSummary %}
+
+- It must be located at the **root** of the project.
+- The only required keys are the `"manifest_version"`, `"name"` and `"version"`.
+- It supports comments (`//`) during development but not in the Chrome Web Store.
+
+{% endDetails %}
+
+### Step 2: Provide the icons {: #step-2 }
+
+So, why do we need icons? Although [icons][doc-icons] are optional during development, they are
+required if you plan to distribute your extension on the Chrome Web Store. They also appear in other
+places like the Extension Management page.
+
+Create an `images` folder and place the icons inside. You can download the icons on
+[Github][github-rt-icons]. Next, add the following code to declare the icons in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "icons": {
+    "16": "images/icon-16.png",
+    "32": "images/icon-32.png",
+    "48": "images/icon-48.png",
+    "128": "images/icon-128.png"
+  }
+  ...
+}
+```
+
+We recommend using PNG files, but other file formats are allowed, except for SVG files. Need help
```

Comment:
The [linked page](https://developer.chrome.com/docs/webstore/images/#icons) states that "The image must be in PNG format." I believe that is incorrect but haven't double checked it myself.

In either case, we should update either this tutorial or the linked page to resolve the potential confusion caused by these contradictory statements.

Reply:
You are right. At this point, I'm leaning towards removing the best practices reference for now. We have a link to the [manifest icons](https://developer.chrome.com/docs/extensions/mv3/manifest/icons/) in a previous paragraph, which should suffice.